packages/cosmic-swingset/go.mod

```diff
@@ -3,39 +3,29 @@ module github.com/Agoric/agoric-sdk/packages/cosmic-swingset
 go 1.14
```

Comment:
WHAT? I didn't change this file. At least not on purpose.

I'll be sure to remove it from this PR. But attn @michaelfig @kriskowal @warner to diagnose what happened?

Reply:
This is the analog of package.json for Go. The Go toolchain may automatically update it any time it deems it necessary, since it’s largely inferred from individual go source files and captures whatever it downloaded to satisfy your dependencies.

Reply:
In any case, Go change gone from this PR.